app/avo/resources/user_resource.rb

```diff
@@ -40,6 +40,7 @@ class UserResource < Avo::BaseResource
       field :mfa_seed, as: :text, visible: ->(_) { false }
       field :mfa_level, as: :select, enum: ::User.mfa_levels
       field :mfa_recovery_codes, as: :text, visible: ->(_) { false }
+      field :hashed_mfa_recovery_codes, as: :text, visible: ->(_) { false }
```

Comment:
do we even need to have those visible?

Reply:
they're not visible, just listed so we can see that the recovery codes have been reset in audits